cmd/cluster-config-operator/main.go

```diff
@@ -25,6 +25,7 @@ func main() {
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 
 	logs.InitLogs()
+	logs.AddFlags(pflag.CommandLine)
```

Comment:
don't mix bumps with changes

Reply:
Hey @sttts , after the bump of `component-base` the way the flags get initialized from `func init` in the logger changed in this [commit](https://github.com/kubernetes/component-base/commit/0c68d203462388c5319702773d799232d01def68) so this is explicitly binding the flags to make sure we get the same behavior, otherwise the logging flags don't get added to `pflag` like they were before.

Reply:
Actually, I missed that this [PR](https://github.com/openshift/cluster-config-operator/pull/229) already updated component base, if it's still desired to have the logger flags be part of the flag parsing, I'll open up a new PR to add that line.